packages/devextreme/js/__internal/ui/chat/avatar.ts

```diff
@@ -99,14 +99,25 @@ class Avatar extends Widget<Properties> {
     return result;
   }
 
-  _getInitials(name: string | undefined): string {
-    if (isDefined(name)) {
-      return String(name).charAt(0).toUpperCase();
+  _getInitials(name: string | number | undefined): string {
```

Comment:
How name can be a number?

Reply:
We maintain this scenario in case the name is given by numbers.

Reply:
During the conversation with Anton, we decided to handle all cases by casting the value to a string. If the value undefined or null, was strictly specified by the user, an empty string is returned. If the user did not specify any value, Unknown User is used by default.

Localization of all text variables is already a separate task.

Reply:
[2c79c4f0b5cd764d261e32aae85f6b4f15127b29]